nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -50,6 +50,12 @@ In addition to numerous new and upgraded packages, this release has the followin
   - This breaks connections to old SSH daemons as ssh-rsa host keys and ssh-rsa public keys that were signed with SHA-1 are disabled by default now
   - These can be re-enabled, see the [OpenSSH changelog](https://www.openssh.com/txt/release-8.8) for details
 
+- ORY Kratos was updated to version 0.8.0-alpha.3
```

Comment:
Do you want to backport this update to release-21.11? Because we already cut the release, so you need to first mark this PR to backport (you can do this by adding the label `backport release-21.11`).

Reply:
Hmm, I guess so... Is it all I need to do? According to [docs](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#backporting-changes), I need to update the title and change the target branch.